frontend/gpay_merchant/lib/location_confirm_on_map_page.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:async';
+import 'package:geocoder/geocoder.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class MapView extends StatefulWidget {
+  MapView({Key key, this.street, this.city, this.latitude, this.longitude}) : super(key: key);
+  String street;
+  String city;
+  double latitude;
+  double longitude;
+  @override
+  State<MapView> createState() => MapViewState();
+}
+
+/*
+ * This class contains the view to confirm the location of the business.
+ * It takes in the initial coordinates and returns the updated ones after
+ * the user has verified them.
+ */
+
+class MapViewState extends State<MapView> {
+  Completer<GoogleMapController> googleMapController = Completer();
+
+  @override
+  Widget build(BuildContext context) {
+    double latitudeAfterUserChange = widget.latitude;
+    double longitudeAfterUserChange = widget.longitude;
+    final CameraPosition initialCameraPosition = CameraPosition(
+    target: LatLng(widget.latitude, widget.longitude),
+    zoom: 19,
+  );
```

Comment:
https://dart.dev/guides/language/effective-dart/style#do-format-your-code-using-dartfmt

Reply:
I am not very sure about what you mean by this comment, could you elaborate?

Reply:
The indentation is a little off here. 
It should be,
final CameraPosition initialCameraPosition = CameraPosition(
    target: LatLng(widget.latitude, widget.longitude),
    zoom: 19,
);
Since it is difficult to fix these manually, you can use the formatter mentioned above which does this job for you.

Reply:
I have fixed it now and also formatted all the pages with the formatter